app/src/main/res/values-de/strings.xml

```diff
@@ -73,19 +73,18 @@
     <string name="aiMenuRegenerate">Eine Antwort neu generieren</string>
     <string name="aiMenuSeriousWriting">Seriöses Schreiben</string>
     <string name="aiMenuShorten">Kürzen</string>
-    <string name="aiPromptAnswer">Sagen Sie dem Assistenten, wie er Ihnen bei der Beantwortung dieser E-Mail helfen kann.</string>
+    <string name="aiPromptAnswer">Erklären Sie Euria, wie sie Hilfe bei der Beantwortung dieser E-Mail erhält.</string>
     <string name="aiPromptExample1">eine Dankesmail für ein Willkommensgeschenk</string>
     <string name="aiPromptExample2">eine E-Mail mit der Bestätigung der Hotelbuchung</string>
     <string name="aiPromptExample3">eine Dankesmail für eine Einladung zu einer Veranstaltung</string>
     <string name="aiPromptExample4">eine Einladungsmail zu einer Firmenveranstaltung</string>
     <string name="aiPromptExample5">eine E-Mail mit der Bitte um einen Termin mit einem wichtigen Kunden</string>
-    <string name="aiPromptGenerationLoader">Der Assistent schreibt…</string>
-    <string name="aiPromptPlaceholder">Sagen Sie dem Schreibassistenten, was Sie schreiben wollen. Zum Beispiel: \"%s\".</string>
-    <string name="aiPromptTag">Beta</string>
-    <string name="aiPromptTitle">Schreibassistent</string>
+    <string name="aiPromptGenerationLoader">Euria schreibt…</string>
+    <string name="aiPromptPlaceholder">Sagen Sie Euria, was Sie schreiben wollen. Zum Beispiel \"%s\".</string>
+    <string name="aiPromptTitle">Euria</string>
     <string name="aiPromptValidateButton">Erzeugen</string>
     <string name="aiReplaceSubjectDescription">Der Betreff wird zu:\n\n%s</string>
-    <string name="aiReplaceSubjectTitle">Ersetzen Sie der aktuelle Betreff durch den Vorschlag des Assistenten?</string>
+    <string name="aiReplaceSubjectTitle">Das aktuelle Objekt durch Eurias Vorschlag ersetzen?</string>
```

Comment:
Put back "Betreff" into the translation, "Objekt" is not the correct translation for an "email subject" in german

Reply:
The gender of the word is wrong now, it should be masculine not neutral